.goreleaser.yaml

```diff
@@ -0,0 +1,55 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    - go mod tidy
```

Comment:
Do we need it if we run it in a diff CI action?

Reply:
I assume its from the default `goreleaser init` setup, but might not be needed.

Reply:
This is also for when running locally, not just a CI config.